python/rtdb2.py

```diff
@@ -6,7 +6,7 @@
 import datetime, time
 import os
 
-RTDB2_DEFAULT_PATH = "/tmp/rtdb2_storage"
+RTDB2_DEFAULT_PATH = "/tmp/rtdb2_storage/2/default"
```

Comment:
Similar change seems to required in alias file.
Tools only work when agent 2 is running.

Reply:
This should be fixed now